tools/pangolin/pangolin.xml

```diff
@@ -6,7 +6,7 @@
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
 
-pangolin '$input1' -t "\${GALAXY_SLOTS:-1}" --max-ambig '$additional.max_ambig' --min-length '$additional.min_length' $additional.write_tree &&
+pangolin '$input1' -t "\${GALAXY_SLOTS:-1}" --max-ambig '$additional.max_ambig' --min-length '$additional.min_length' $additional.write_tree $additional.include_putative &&
```

Comment:
Have you tested this tool when `include_putative` is true and false? Reason being is that some scripts may not like having a statement like this `myscript -a 1 -b 2 '' -c 5`, where there are `''` in the middle of the command.

Reply:
Yep it works both ways! No `' '` in the input at all as those input parameters aren't surrounded with `' '`

Reply:
Oh interesting, was developing a wrapper and had `falsevalue=""` and it inserted a `''` in my command line statement.